website/pages/en/index.js

```diff
@@ -538,9 +538,9 @@ class Index extends React.Component {
                   <MarkdownBlock>
                     <translate>
                       A lot of people! With
-                      [8.5m](https://www.npmjs.com/package/jest) downloads in
+                      [4m](https://www.npmjs.com/package/jest) downloads in
```

Comment:
4m in the last 7 days, should multiply by 4 if monthly 🙂

Reply:
:+1: